package.json

```diff
@@ -77,9 +77,9 @@
     "@react-native-community/cli-platform-ios": ">=4.10.0",
     "mustache": "^4.0.0",
     "react": "~16.8.6 || ~16.9.0 || ~16.11.0 || ~16.13.1 || ~17.0.1 || ~17.0.2",
-    "react-native": "^0.0.0-0 || ^0.60 || ^0.61 || ^0.62 || ^0.63 || ^0.64 || ^0.65 || ^0.66 || 1000.0.0",
-    "react-native-macos": "^0.0.0-0 || ^0.60 || ^0.61 || ^0.62 || ^0.63 || ^0.64",
-    "react-native-windows": "^0.0.0-0 || ^0.62 || ^0.63 || ^0.64 || ^0.65 || ^0.66"
+    "react-native": "^0.0.0-0 || 0.60 - 0.67 || 1000.0.0",
```

Comment:
TIL I learned you can use `-` for a range 🤣

Reply:
The string was getting really long so I played around with https://semver.npmjs.com/ to see if there's anything I can do to fix it. Then I saw the boxes at the bottom…